openmoc/plotter.py

```diff
@@ -1015,10 +1015,10 @@ def plot_spatial_data(domains_to_data, plot_params, get_figure=False):
     Examples
     --------
     A user may invoke this function from an Python file as follows:
```

Comment:
Sure. If in looking through this you find an other issues feel free to point them out and I'll be sure to update them.